deployment/artifacts/mainnet.json

```diff
@@ -1581,375 +1581,6 @@
         }
     },
     "137": {
-        "BetaProgramInitiator": {
```

Comment:
Will this cause conflicts with #298 ? or are they based over each other?

Reply:
✔️  I reorganized to base this PR over #298 (merged)